distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc

```diff
@@ -35,12 +35,12 @@ If console access is needed while running as a service, run the `client` script:
 
 .*NIX
 ----
-<${branding}_HOME/bin/client
+<${branding}_HOME>/bin/client
 ----
 
 .Windows
 ----
-<${branding}_HOME>/bin/client.bat
+<${branding}_HOME>/bin/client.bat -h localhost
```

Comment:
Should we add a note that this is an example? Basically we are specifying the host to connect to.